docs/docs/testkube-cloud/articles/status-pages.md

```diff
@@ -0,0 +1,238 @@
+
+## Incidents
+
+Testkube Status Pages streamlines the incident management processes by providing a central platform for incident tracking, communication, and documentation. This is a critical aspect of Testkube Status Pages and involves the communication processes and procedures for detecting, reporting, and resolving issues or incidents that affect the availability or performance of your software or services. This helps teams respond to incidents more efficiently, maintain transparency with stakeholders, and continuously improve their incident response procedures to ensure the reliability of their software or services.
+
+Incidents are simple objects responsible for communication between the service providers and their users. Managing them is as easy as creating them via the Status Pages management page.
+
+![incidents-edit](../../img/incidents-edit.png)
+
+They will be shown at the bottom, similar to a news feed.
+
+![incidents-main](../../img/incidents-main.png)
+
+On the top of each individual incident there is the start date. The incidents are ordered by this, with the latest incident on top. The other relevant dates, for example when it was created, updated and resolved are shown at the very bottom. The end date can be left intentionally empty, signifying that this incident has not been resolved.
+
+The title of the incident will be colored based on the configured severity:
+
+* critical: red
+* major: red
+* minor: orange
+* low: yellow
+* info: white
+
+The interpretation of the severity is up to the decision of the team, just make sure everyone is well-informed and has agreed to it.
+
+Incidents can have three different states depending on the visibility: draft, published and archived. Only published incidents are visible in this view.
+
+## Collaboration and Access Control
+
+Status Pages can be either private or public. Public status pages are published to the internet. Private pages can be shared with anyone in the same organization having read access to the environment.
+
+## Best Practices
+
+Best practices are essential for effectively using Testkube Status Pages to communicate the status of your software projects. These practices help ensure that your status pages are informative, reliable, and serve their intended purpose. Here are some best practices for Testkube Status Pages:
+
+### Relevant Tests
+
+The very first thing you have to consider, even before thinking about your status page, is the type of tests you are creating. You have to make sure that there are tests that are monitoring the not just the behavior of a service, but also the availability. For this, Kubernetes is using liveness probes, which check periodically if your service is still responsive on a user-configured endpoint. In case your application is already running in Kubernetes, consider reusing this endpoint in Testkube.
```

Comment:
```suggestion
The very first thing you have to consider, even before thinking about your status page, is the type of tests you are creating. You have to make sure that there are tests that are monitoring not just the behavior of a service, but also the availability. For this, Kubernetes is using liveness probes, which check periodically if your service is still responsive on a user-configured endpoint. In case your application is already running in Kubernetes, consider reusing this endpoint in Testkube.
```

Reply:
updated, thanks!